### PR TITLE
Feat: changed login signup style.

### PR DESCRIPTION
**Pull request by zoef-yt**
changed login signup style.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,8 +13,8 @@
 				<a href="index.html"> <h1>Need Games</h1> </a>
 			</div>
 			<input type="search" class="text-field" />
-			<div class="navbar-cta"
-				><a href="./screens/login-page/login-page.html" class="btn btn-link">Hello, Login</a>
+			<div class="navbar-cta">
+				<a onclick="openModal('login-modal')" id="login-text" class="btn btn-link">Hello, Login</a>
```
Comment (iRohitGaur) [inline index.html]: suggestion: don’t use onclick in html. implement this on the js side.

**Review by iRohitGaur — APPROVED** [review]
clean and understandable code. The UI looks awesome. Great work. Good to merge 👍🏻

File: index.html (hunk 1 above)
Comment (archit7448) [reply index.html]: +1 It creates problem

**Review by archit7448 — APPROVED** [review]
good to merge

File: screens/login-page/login-page.css [hunk 2]
```diff
@@ -14,22 +14,41 @@
...
 	display: flex;
 	flex-direction: row;
 	align-items: center;
+	justify-content: space-between;
 }
 
 .checkbox {
 	cursor: pointer;
-	margin: 0 1rem 0 0;
+	margin: 0 0.5rem 0 0;
```
Comment (zoyatanweer) [inline screens/login-page/login-page.css:37]: 5px maybe?

**Review by zoyatanweer — APPROVED** [review]

**Review by sudipkundu999 — CHANGES_REQUESTED** [review]
UI looks great!
The position of badges look a bit off
zoef

**Comment by zoef-yt** [comment]
> UI looks great! The position of badges look a bit off zoef

Yeah actually this is coming from my component library that's why it is like that will be fixing it in CL it will be fixed it automatically.

**Review by sudipkundu999 — APPROVED** [review]

**Review by j24m — APPROVED** [review]
Perfection!!! GTM. 💯

**Pull request merged**